modules/localgov_roles/localgov_roles.install

```diff
@@ -13,4 +15,28 @@ function localgov_roles_install() {
   foreach ($roles as $role => $permissions) {
     user_role_grant_permissions($role, $permissions);
   }
+  _remove_access_files_overview_permission();
+}
+
+/**
+ * Removes the 'access files overview' permission from all roles.
+ *
+ * If a particular role needs this permission, it should be added via the
+ * permissions page.
+ */
+function _remove_access_files_overview_permission() {
+  $roles = Role::loadMultiple();
+  foreach ($roles as $role) {
+    if ($role->hasPermission('access files overview')) {
+      $role->revokePermission('access files overview');
+      $role->save();
+    }
+  }
+}
```

Comment:
I'm not so sure of this, if there are custom roles that have been assigned this then they will get deleted. Sometimes it is useful if there are files which for what ever reason are not yet in media. I'd limit this to localgov drupal provided roles, and I'd check if they have been altered from default. Noting the update hook also runs this.